Describe the Layout Updates Options columns for the "Using a Widget to Position a Block" page (#1341)
magento/merchdocs: src/cms/widget-static-block.md

   ![]({% link images/images/widget-layout-update-home-page.png %}){: .zoom}
   _Widget layout updates_

> ### Layout Updates Options

Reviewer: ```suggestion
### Layout update options
```

{: .fields-table }
|**_Categories_**||

> |Anchor Categories|Displays the widget on the anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|Anchor Categories|Displays the widget on the anchor category page. Options:<br/>**Categories** - Categories where the anchor is displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

> |Non-Anchor Categories|Displays the widget on the non-anchor category page. Options:<br/>**Categories** - Categories where the anchor will be displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|Non-Anchor Categories|Displays the widget on the non-anchor category page. Options:<br/>**Categories** - Categories where the anchor is displayed. Options: All / Specific Categories<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

|**_Products_**||

> |All Product Types|Displays the widget on either a specific type of product page, or on all product pages. Options:<br/>**Products** - Products for which the widget is displayed. Options: All / Specific Products<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|All Product Types|Displays the widget on either a specific type of product page, or on all product pages. Options:<br/>**Products** - Products for which the widget is displayed. Options: All / Specific Products<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

|**_Generic Pages_**||

> |All Pages|Displays the widget on all pages. Options:<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|All Pages|Displays the widget on all pages. Options:<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

> |Specified Page|Displays the widget on a specific page. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|Specified Page|Displays the widget on a specific page. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```

> |Page Layouts|Displays the widget on pages with a certain layout. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where the widget is to appear.<br/>**Template** - Determines the theme of the layout.|

Reviewer: ```suggestion
|Page Layouts|Displays the widget on pages with a certain layout. Options:<br/>**Page** - Pages for which the widget is displayed.<br/>**Container** - Set container to the part of the page layout where you want to display the widget.<br/>**Template** - Determines the theme of the layout.|
```
